### PR TITLE
Serve static files from commento binary

### PR DESCRIPTION
**Pull request by smbkr**
Allows the static assets (js, css) to be served by the Go binary itself. 

This would change the example HTML to something like the following:
```<html>
<body>
<script src="http://0.0.0.0:8080/assets/commento.min.js"></script>
</body>

<script>
    init_commento("http://0.0.0.0:8080");
</script>

<div id="commento">
</div>
</html>```

### CODE REVIEW

File: vendor/commento.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ init_commento = function(server) {
     COMMENTO_SERVER = server;
     loadJS("https://cdn.rawgit.com/showdownjs/showdown/1.6.3/dist/showdown.min.js", function() {
         loadCSS("https://cdn.rawgit.com/picturepan2/spectre/master/docs/dist/spectre.min.css");
-        loadCSS("https://cdn.rawgit.com/adtac/commento/0.0.2/vendor/commento.min.css");
+        loadCSS(server + "/assets/commento.min.css");
```
Comment (adtac) [inline vendor/commento.js]: `vendor/commento.min.css`, no? Or should we rename the `vendor` directory to `assets`?

Comment (adtac) [inline vendor/commento.js]: Ah nevermind, I just noticed the other change :+1:

**Comment by adtac** [comment]
@smbkr this looks great (just tested it locally). Could you add another commit changing the README to reflect this and push? Thanks!

**Comment by smbkr** [comment]
Sure, will do. With regards to your other comment, I could also rename the vendor/ dir to assets/, so that the layout of the project is the same as the http routes? 

The reason I went with /assets/ instead of /vendor/ for the routing is because it seemed a better fit semantically to me, since "vendor" is typically used in a dependency context.

**Comment by adtac** [comment]
I think it's probably a good idea to rename the directory to `assets`. As you said, `vendor` doesn't make much sense here.

**Comment by firstrow** [comment]
@smbkr agree! vendor dir should be only used for golang vendoring.

**Comment by adtac** [comment]
Or `/static/` works too.

**Comment by smbkr** [comment]
Whoomp, there it is.

**Comment by adtac** [comment]
@smbkr This is ready to merge, but it'll have a few merge conflicts. So I'll cherry pick this (and a few other PRs) into a single mega PR (you'll still be the author of your commits) and merge them, if that's okay with you :)

**Comment by adtac** [comment]
Cherry-picked and merged here: https://github.com/adtac/commento/pull/35

Of course, he commit author information is unchanged.

Thanks!

**Pull request closed without merging**